docs/workshop/examples/42_otel_jaeger/README.md

```diff
@@ -0,0 +1,134 @@
+# CNCF OpenTelemetry Jaeger Interworking
+
```

Comment:
we used to add a link to the code on github like:

```md
!!! note
    All the application code here is available from the docs [git repository](https://github.com/tremor-rs/tremor-www-docs/tree/main/docs/workshop/examples/42_otel_jaeger).
```

Reply:
Added note - good catch